src/app/src/user/directives/UserActionsDropdown.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+function UserActionsDropdown($dropdown) {
+    return {
+        scope: {userActionsDropdown: '=', userId: '='},
+        controller: 'UserDeleteCtrl',
+        controllerAs: 'vm',
+        restrict: 'A',
+        link: function(scope, element, attrs, UserDeleteCtrl) {
+            var dropdown = $dropdown(element, {
+                templateUrl: 'gzero/admin/views/user/directives/userActionsDropdown.tpl.html',
+                animation: 'am-flip-x',
+                placement: 'bottom-right'
+            });
+
+            element.on('click', function() {
+                dropdown.$scope.user = scope.userActionsDropdown;
+                dropdown.$scope.userId = scope.userId; // Pass user id to the view
```

Comment:
You can use record attribiute to pass the entire user record:

` data-record="user"`

Then it will be available in the view:

` dropdown.$scope.record = scope.record; // Pass record to the view`

Where the delete modal is called, with record.id:

`deleteModal.showModal(record.id);`

And used to show the modal widnow for specified user:

`showModal: function(userId)`